www/usecases/customquestions/custom_box_whisker.php

```diff
@@ -30,29 +30,32 @@
     {
       "response_id": "custom-box-whisker-response-1",
       "type": "custom",
-      "js": "//' . $_SERVER['HTTP_HOST'] . '/usecases/customquestions/custom_box_whisker_js.php",
-      "css": "//' . $_SERVER['HTTP_HOST'] . '/usecases/customquestions/custom_box_whisker.css",
       "stimulus": "Draw a <b>box &amp; whisker</b> chart for the following: <b>6, 2, 5, 3, 6, 10, 11, 6</b>",
-      "params_line_min": 0,
-      "params_line_max": 17,
-      "params_step": 0.5,
-      "params_mark_small": 0,
-      "params_mark_big": 1,
-      "params_width": 600,
-      "params_height": 150,
-      "params_range_1": 2,
-      "params_range_2": 14,
-      "params_quartile_1": 4,
-      "params_median": 6,
-      "params_quartile_3": 10,
-      "params_box1_color": "#bbbbbb",
-      "params_box2_color": "#999999",
-      "valid_range_1": 2,
-      "valid_range_2": 11,
-      "valid_quartile_1": 4,
-      "valid_median": 6,
-      "valid_quartile_3": 8.5,
-      "score": 1
+      "js": {
+        "question": "//' . $_SERVER['HTTP_HOST'] . '/usecases/customquestions/custom_box_whisker_q.js",
+        "scorer": "//' . $_SERVER['HTTP_HOST'] . '/usecases/customquestions/custom_box_whisker_s.js"
+      },
+      "css": "//' . $_SERVER['HTTP_HOST'] . '/usecases/customquestions/custom_box_whisker.css",
+      "line_min": 1,
+      "line_max": 19,
+      "step": 1,
+      "min": 2,
+      "max": 14,
+      "quartile_1": 4,
+      "median": 6,
+      "quartile_3": 10,
+      "score": 1,
+      "valid_response": {
+        "type": "object",
+        "value": {
+            "min": 2,
+            "max": 11,
+            "quartile_1": 4,
+            "median": 6,
+            "quartile_3": 8
+        }
+      },
+      "instant_feedback": true
     }
   ],
     "beta_flags": {
```

Comment:
@gonzalozawa I would remove this.

Reply:
@michaelsharman do you mean the entire custom_box_whisker.php demo and only keep the Authoring one? Please confirm. Thanks!

Reply:
As discussed, I've removed the "beta_flags" from Custom Questions demos.